### PR TITLE
[SNDREC32] French translation update

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

- Completion of FR-fr translation for SNDREC32 module.

Note : Caution when merged along with https://github.com/reactos/reactos/pull/2754

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-french

**Comment by KyleKatarn31** [comment]
> 
> 
> Cc @reactos/lang-french

This goes to an Error 404 page.
If it's the link for the Fr-fr translator team, any possibility to get added there ?

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 1]
```diff
@@ -25,35 +25,35 @@ BEGIN
...
         MENUITEM "Propriétés", ID_FILE_PROPERTIES
         MENUITEM SEPARATOR
         MENUITEM "Sortir", ID_FILE_EXIT
     END
     POPUP "&Edition"
     BEGIN
         MENUITEM "&Copie", ID_EDIT_COPY
-        MENUITEM "&Paste Insert", ID_EDIT_PASTE, GRAYED
-        MENUITEM "Paste Mi&x", ID_EDIT_PASTEMIX, GRAYED
+        MENUITEM "Coller &inésrer", ID_EDIT_PASTE, GRAYED
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:36]:
```suggestion
        MENUITEM "Coller &Insérer", ID_EDIT_PASTE, GRAYED
```

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 2]
```diff
@@ -25,35 +25,35 @@ BEGIN
...
+        MENUITEM "Coller &mélanger", ID_EDIT_PASTEMIX, GRAYED
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:37]:
```suggestion
        MENUITEM "Coller &Mélanger", ID_EDIT_PASTEMIX, GRAYED
```

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 3]
```diff
@@ -25,35 +25,35 @@ BEGIN
...
         MENUITEM "Enregistrer sous...", ID_FILE_SAVEAS, GRAYED
-        MENUITEM "Restorer...", ID_FILE_RESTORE, GRAYED
+        MENUITEM "Restaurer...", ID_FILE_RESTORE, GRAYED
         MENUITEM "Propriétés", ID_FILE_PROPERTIES
         MENUITEM SEPARATOR
         MENUITEM "Sortir", ID_FILE_EXIT
     END
     POPUP "&Edition"
     BEGIN
         MENUITEM "&Copie", ID_EDIT_COPY
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:35]:
```suggestion
        MENUITEM "&Copier", ID_EDIT_COPY
```

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 4]
```diff
@@ -25,35 +25,35 @@ BEGIN
...
-        MENUITEM "&Paste Insert", ID_EDIT_PASTE, GRAYED
-        MENUITEM "Paste Mi&x", ID_EDIT_PASTEMIX, GRAYED
+        MENUITEM "Coller &inésrer", ID_EDIT_PASTE, GRAYED
+        MENUITEM "Coller &mélanger", ID_EDIT_PASTEMIX, GRAYED
         MENUITEM SEPARATOR
         MENUITEM "&Insérer un fichier...", ID_EDIT_INSERTFILE
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:39]:
```suggestion
        MENUITEM "Insérer un &fichier...", ID_EDIT_INSERTFILE
```

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 5]
```diff
@@ -25,35 +25,35 @@ BEGIN
...
-        MENUITEM "&Mix with File...", ID_EDIT_MIXFILE
+        MENUITEM "&Mélanger avec un fichier...", ID_EDIT_MIXFILE
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:40]:
```suggestion
        MENUITEM "Mé&langer avec un fichier...", ID_EDIT_MIXFILE
```

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 6]
```diff
@@ -25,35 +25,35 @@ BEGIN
...
     POPUP "Effet&s"
     BEGIN
-        MENUITEM "&Increase Volume (by 25%)", ID_EFFECTS_INCVOL
-        MENUITEM "&Decrease Volume", ID_EFFECTS_DECVOL
+        MENUITEM "&Augmenter le volume (de 25%)", ID_EFFECTS_INCVOL
+        MENUITEM "&Réduire le volume", ID_EFFECTS_DECVOL
         MENUITEM SEPARATOR
-        MENUITEM "&Increase Speed (by 100%)", ID_EFFECTS_INCSPD
-        MENUITEM "&Decrease Speed", ID_EFFECTS_DECSPD
+        MENUITEM "&Augmenter la vitesse (de 100%)", ID_EFFECTS_INCSPD
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:52]:
```suggestion
        MENUITEM "Aug&menter la vitesse (de 100%)", ID_EFFECTS_INCSPD
```

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 7]
```diff
@@ -25,35 +25,35 @@ BEGIN
...
+        MENUITEM "&Réduire la vitesse", ID_EFFECTS_DECSPD
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:53]:
```suggestion
        MENUITEM "Réd&uire la vitesse", ID_EFFECTS_DECSPD
```

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 8]
```diff
@@ -25,35 +25,35 @@ BEGIN
...
         MENUITEM SEPARATOR
-        MENUITEM "&Add Echo", ID_EFFECTS_ECHO
-        MENUITEM "&Reverse", ID_EFFECTS_REVERSE
+        MENUITEM "&Ajouter de l'écho", ID_EFFECTS_ECHO
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:55]:
```suggestion
        MENUITEM "Aj&outer de l'écho", ID_EFFECTS_ECHO
```

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 9]
```diff
@@ -25,35 +25,35 @@ BEGIN
         MENUITEM "Ouvrir...", ID_FILE_OPEN
         MENUITEM "Enregistrer", ID_FILE_SAVE, GRAYED
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:26]:
```suggestion
        MENUITEM "&Enregistrer", ID_FILE_SAVE, GRAYED
```

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 10]
```diff
@@ -25,35 +25,35 @@ BEGIN
...
         MENUITEM "Enregistrer sous...", ID_FILE_SAVEAS, GRAYED
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:27]:
```suggestion
        MENUITEM "Enregistrer &sous...", ID_FILE_SAVEAS, GRAYED
```

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 11]
```diff
@@ -25,35 +25,35 @@ BEGIN
...
-        MENUITEM "Restorer...", ID_FILE_RESTORE, GRAYED
+        MENUITEM "Restaurer...", ID_FILE_RESTORE, GRAYED
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:28]:
```suggestion
        MENUITEM "&Restaurer...", ID_FILE_RESTORE, GRAYED
```

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 12]
```diff
@@ -25,35 +25,35 @@ BEGIN
...
         MENUITEM "Propriétés", ID_FILE_PROPERTIES
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:29]:
```suggestion
        MENUITEM "&Propriétés", ID_FILE_PROPERTIES
```

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 13]
```diff
@@ -25,35 +25,35 @@ BEGIN
...
         MENUITEM SEPARATOR
         MENUITEM "Sortir", ID_FILE_EXIT
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:31]:
```suggestion
        MENUITEM "&Quitter", ID_FILE_EXIT
```

File: base/applications/sndrec32/lang/fr-FR.rc [hunk 14]
```diff
@@ -25,35 +25,35 @@ BEGIN
         MENUITEM "Ouvrir...", ID_FILE_OPEN
```
Comment (HBelusca) [inline base/applications/sndrec32/lang/fr-FR.rc:25]:
```suggestion
        MENUITEM "&Ouvrir...", ID_FILE_OPEN
```

**Comment by KyleKatarn31** [comment]
Thanks ! Conflict in https://github.com/reactos/reactos/pull/2754/ cleared.

**Pull request merged**